users/templates/registration/password_change_form.html

```diff
@@ -1,5 +1,50 @@
+{% extends "base.html" %}
+{% load i18n static %}
+
 {% block content %}
+<div id="content-main">
+
+  <div class="row">
+    <div class="col s8 offset-s1 offset-m2 offset-l3">
+      <p>{% trans "Please enter your old password, for security's sake, and then enter your new password twice so we can verify you typed it in correctly." %}</p>
```

Comment:
to można przerzucić w blocktransy: https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#std:templatetag-blocktrans